pytransform3d/transform_manager/_temporal_transform_manager.py

```diff
@@ -0,0 +1,209 @@
+import abc
+
+import numpy as np
+
+from ..batch_rotations import norm_vectors
+
+from ._transform_graph_base import TransformGraphBase
+from ..transformations import check_transform, transform_from_pq, \
+    dual_quaternion_from_pq, pq_from_dual_quaternion, dual_quaternion_sclerp
+
+
+class TimeVaryingTransform(abc.ABC):
+    """Time-varying rigid transformation."""
```

Comment:
Maybe we can add "This class shall be inherited from to use the `TemporalTransformManager`, refer to NumpyTransform implementation" or something similar.

This text appears in the docs, so people browsing around will see what this class is meant for.